src/v2/Apps/Order/Routes/Shipping/index.tsx

```diff
@@ -186,18 +187,24 @@ export class ShippingRoute extends Component<ShippingProps, ShippingState> {
     const artsyShippingInternational = !!this.getOrderArtwork()
       ?.artsyShippingInternational
 
+    const artwork = this.getOrderArtwork()
     const shippingCountry = this.isCreateNewAddress()
       ? this.state.address.country
       : addresses &&
         addresses.find(
           address => address?.node?.internalID == this.state.selectedAddressID
         )?.node?.country
 
+    const isDomesticOrder =
+      (COUNTRIES_IN_EUROPEAN_UNION.includes(shippingCountry) &&
+        COUNTRIES_IN_EUROPEAN_UNION.includes(artwork?.shippingCountry)) ||
+      artwork?.shippingCountry == shippingCountry
+    const isInternationalOrder = !isDomesticOrder
+
     return (
       this.state.shippingOption === "SHIP" &&
-      ((processWithArtsyShippingDomestic && shippingCountry === "US") ||
-        (artsyShippingInternational && shippingCountry !== "US"))
-      // we are only allowing US based Artsy Shipping for now
+      ((processWithArtsyShippingDomestic && isDomesticOrder) ||
+        (artsyShippingInternational && isInternationalOrder))
     )
```

Comment:
Discussed with @leamotta, we dont need to wire in the feature flag from unleash in force as Volt/gravity are protecting us from having users opt works into artsy shipping from the new countries 

https://artsy.slack.com/archives/C9ZJYFDNF/p1655996716769069